utilities/pipelines/staticValidation/compliance/helper/child-module-publish-allowed-list.json

```diff
@@ -3,11 +3,27 @@
     "avm/res/authorization/role-assignment/mg-scope",
     "avm/res/authorization/role-assignment/sub-scope",
     "avm/res/authorization/role-assignment/rg-scope",
-    "avm/res/storage/storage-account/blob-service/container",
-    "avm/res/storage/storage-account/file-service/share",
-    "avm/res/network/virtual-network/subnet",
+    "avm/res/event-hub/namespace/eventhub",
     "avm/res/key-vault/vault/access-policy",
     "avm/res/key-vault/vault/key",
-    "avm/res/key-vault/vault/secret"
+    "avm/res/key-vault/vault/secret",
+    "avm/res/network/private-dns-zone/a",
+    "avm/res/network/private-dns-zone/aaaa",
+    "avm/res/network/private-dns-zone/cname",
+    "avm/res/network/private-dns-zone/mx",
+    "avm/res/network/private-dns-zone/ptr",
+    "avm/res/network/private-dns-zone/soa",
+    "avm/res/network/private-dns-zone/srv",
+    "avm/res/network/private-dns-zone/txt",
+    "avm/res/network/virtual-network/subnet",
+    "avm/res/storage/storage-account/blob-service/container",
```

Comment:
Thanks for this. Can I ask to remove storage children from the list? I'd need them for an update and hence assigned those corresponding tasks to me. 
In general I'd recommend to create one PR per each parent.

Reply:
You're 100% right. When looking at the list of issues I didn't see the difference of the icon. That's my bad and I'll revert the Storage code.

On the 2nd note: I opted for consolidating the publishing into a single PR as it's less prone to conflicts of the `child-module-publish-allowed-list.json` file. 
Anyways, no other module would be added to this one as the other remaining one (API) is too big to be reasonably reviewed.